wildfly-standalone-centos/README.md

```diff
@@ -22,72 +22,77 @@
 <!-- /TOC -->
 
 ## Solution Overview
-WildFly 18 is the latest release in a series of JBoss open-source application server offerings. WildFly 18 is an exceptionally fast, lightweight and powerful implementation of the Jakarta Platform specifications. The state-of-the-art architecture built on the Modular Service Container enables services on-demand when your application requires them.
 
-This Azure quickstart template deploys a web applicaton deployed on WildFly 18.0.1.Final running on CentOS 8.
+This template creates a standalone node of WildFly on CentOS to a Standard F1 VM size in your Resource Group (RG) which includes a Public DNS, Private Virtual Network and security configuration. It is ideal for development and testing of enterprise Java applications on Azure. 
 
-To obtain a rhsm account go to: www.redhat.com and sign in.
+WildFly is an open-source application server runtime that is exceptionally fast, flexible, lightweight and powerful. It is Jakarta EE 8 Full Platform and Web Profile compatible.  The state-of-the-art architecture built on the Modular Service Container enables services on-demand when your application requires them.
+
+This Azure quickstart template deploys a web applicaton on WildFly 18.0.1.Final running on CentOS 8.
+
+The technology behind WildFly is also available in Red Hat JBoss Enterprise Application Platform (EAP), a hardended enterprise subscription with Red Hat's world-class support, long multi-year maintenance cyccles and exclusive content. JBoss EAP is an open-soure application server that you can download for free, for developement use.  To obtain a Red Hat Subscription Management (RHSM) account for JBoss EAP, go to: www.redhat.com 
 
 ## Template Solution Architecture 
 
-This template creates all of the compute resources to run WILDFLY 18.0.1 on top of CentOS 8.0, deploying the following components:
+This template creates all of the Azure compute resources to run WildFly 18.0.1.Final on top of CentOS 8.0. The following resources are created by this template:
```

Comment:
I have added the term "VM" after CentOS 8.0, please let me know if I should remove it.